tests/tracking/test_client.py

```diff
@@ -731,3 +731,12 @@ def test_delete_model_version_tag(mock_registry_store_with_get_latest_version):
     # delete_model_version_tag with version and stage not set
     with pytest.raises(MlflowException, match="version or stage must be set"):
         MlflowClient().delete_model_version_tag("model_name", key="tag1")
+
+
+def test_update_run(mock_store):
+    MlflowClient().update_run(run_id="run_id", status="FINISHED", name="my name")
+
+    _, kwargs = mock_store.update_run_info.call_args
+    assert kwargs["run_id"] == "run_id"
+    assert kwargs["run_status"] == RunStatus.from_string("FINISHED")
+    assert kwargs["run_name"] == "my name"
```

Comment:
```suggestion
    _, kwargs = mock_store.update_run_info.call_args
    mock_store.assert_called_once_with(
        run_id="run_id", status=RunStatus.from_string("FINISHED"), name="my name"
    )
```

We can use `assert_called_once_with` here.

Reply:
I think `assert_called_once_with` needs to match all arguments so I would need do this:
```python
    mock_store.update_run_info.assert_called_once_with(
        run_id="run_id",
        run_status=RunStatus.from_string("FINISHED"),
        end_time=mock_store.get_run().info.end_time,
        run_name="my name",
    )
```

Does it look good?

Reply:
For `end_time`, we can use `mock.ANY`.

Reply:
good point, i updated it